sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/models/SchemaProperties.java

```diff
@@ -3,38 +3,31 @@
 
 package com.azure.data.schemaregistry.models;
 
+import com.azure.core.annotation.Immutable;
 import com.azure.data.schemaregistry.SchemaRegistryAsyncClient;
 import com.azure.data.schemaregistry.SchemaRegistryClient;
 
-import java.util.Arrays;
-
 /**
- * Stores all relevant information returned from {@link SchemaRegistryClient}/{@link SchemaRegistryAsyncClient} layer.
+ * Stores properties of a schema stored in Schema Registry.
+ *
+ * @see SchemaRegistryAsyncClient
+ * @see SchemaRegistryClient
  */
+@Immutable
 public final class SchemaProperties {
 
     private final String schemaId;
     private final SchemaFormat schemaFormat;
-    private final byte[] schemaBytes;
-    private final String schemaName;
 
     /**
-     * Initializes SchemaRegistryObject instance.
+     * Initializes a new instance.
      *
-     * @param schemaId the schema id
-     * @param schemaFormat type of schema, e.g. avro, json
-     * @param schemaName name of the schema.
```

Comment:
Why did we remove the schemaName?

Reply:
AFAIK, none of the other languages had this parameter. It used to always return null, and I had to add some regex magic to parse it from the returned headers.
ex: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaProperties.cs

Reply:
This might be worth asking the service team to include schemaName in their response.

Reply:
@deyaaeldeen . IIRC, we asked the service team about this?

Reply:
I do not believe but we can discuss this in our next sync. For now, all languages do not have it so let's remove it in Java as well.